corebehrt/main/xgboost_cv.py

```
@@ -0,0 +1,71 @@
+from os.path import join
+from corebehrt.functional.setup.args import get_args
+from corebehrt.modules.preparation.dataset import PatientDataset
+from corebehrt.modules.setup.config import load_config
+from corebehrt.modules.setup.directory import DirectoryPreparer
+from corebehrt.main.helper.finetune_cv import check_for_overlap
+from corebehrt.main.helper.xgboost_cv import cv_loop
+from corebehrt.functional.io_operations.load import load_vocabulary
+
+
+CONFIG_PATH = "./corebehrt/configs/xgboost.yaml"
+from corebehrt.constants.paths import (
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)